docs/version3-0.md

```diff
@@ -12,14 +12,30 @@ Yes! The Action Scheduler Custom Tables plugin code is now part of Action Schedu
 
 By default, Action Scheduler will only initiate a migration from the internal `WPPostStore` data store. To enable migration from any custom datastore add the following filter `add_filter( 'action_scheduler_migrate_data_store', '__return_true' );`.
 
-### I'm currently on PHP <5.5. When I update PHP will the migration start automatically?
+### I'm currently on PHP >=5.5. When I update PHP will the migration start automatically?
```

Comment:
This change is imo incorrect.
The initial question I raised:
> When I'm on PHP 5.4, what will happen when I will upgrade PHP to a newer version (5.5+)?

Right now `I'm currently on PHP >=5.5.` doesn't make sense, because that's the default behavior of AS lib to start migration on PHP 5.5, because that's the requirement for custom tables.